### PR TITLE
Fix metric check

### PR DESCRIPTION
**Pull request by phoppe93**
#### General

- [ ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass 

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

**Comment by phoppe93** [comment]
@sstarcher please merge my commit. 

metrics for elb asg elasticache 
- i add at the the option to use a spicific autoscalinggroup or loadbalancer.

CheckAsgInstancesCreated
- add the option to use a time where in which range the instances are terminated

All metric and checks
- using scheme for variable output name

**Comment by phoppe93** [comment]
@sstarcher please merge ?

**Comment by sstarcher** [comment]
merge conflicts, backwards incompatible changes

File: bin/check-asg-instances-created.rb [hunk 1]
```diff
@@ -34,18 +34,18 @@
...
          short:       '-r AWS_REGION',
          long:        '--aws-region REGION',
          description: 'AWS Region (defaults to us-east-1).',
          default:     ENV['AWS_REGION']
 
-  option :asg_group_name,
-         short:       '-g G',
-         long:        '--asg_group_name AutoScalingGroupName',
-         description: 'AutoScalingGroupName to check'
+  option :asgname,
```
Comment (sstarcher) [inline bin/check-asg-instances-created.rb]: this breaks backward compatibility

Comment (phoppe93) [inline bin/check-asg-instances-created.rb]: reference:
https://github.com/phoppe93/sensu-plugins-aws/blob/master/bin/check-elb-health-fog.rb

better to use the option -n like in the reference.

@sstarcher

**Comment by obazoud** [comment]
You should split this PR in 3 parts.

**Comment by obazoud** [comment]
is it a partial rollback of #171 ?

**Comment by multani** [comment]
What needs to be done to merge this PR?

Most of the --scheme options just don't work for the metrics-* scripts (tested on the -rds, -elb and -asg variants). The metrics-asg.rb script doesn't work because the name of the dimension (AutoScaling) is not right.

**Pull request closed without merging**